.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can you remove this file from this PR? I agree with the spirit...I'd just reco a separate PR that focused on adding prettier support.